### PR TITLE
Feature/impact component

### PR DESCRIPTION
**Pull request by Meghag5**

### CODE REVIEW

**Comment by Meghag5** [comment]
changes made

File: src/Components/Impact.js [hunk 1]
```diff
@@ -1,13 +1,46 @@
 import React, { Component } from "react";
-import ImpactList from "./ImpactList";
+import ImpactCard from "./ImpactCard";
+import styles from "./Css/Impact.module.css";
```
Comment (SinghVikram97) [inline src/Components/Impact.js]: It should be **../css**

**Review by SinghVikram97 — CHANGES_REQUESTED** [review]
Make it responsive according to different screen sizes. Also, make impact cards a little bigger as can be seen in the design. Add <hr/> Above the component also, refer to the design file.

**Comment by Meghag5** [comment]
new changes made

**Pull request closed without merging**